mobile/pubspec.yaml

```diff
@@ -31,6 +32,8 @@ dependencies:
   intl: ^0.19.0
   json_annotation: ^4.9.0
   just_audio: ^0.9.38
+  # TODO: add later if needed (e.g. to fix #31)
+  # just_audio_background: ^0.0.1-beta.13
```

Comment:
This shouldn't be necessary since we're using `audio_service` which should have functionalities of `just_audio_background` built in, but yeah, let's leave it here, maybe it will come handy at some point.